cli.js

```diff
@@ -4,9 +4,14 @@ const exec = require('child_process').exec;
 const InitCommand = require('./cli-commands/init');
 const TestCommand = require('./cli-commands/test');
 const ShapeCommand = require('./cli-commands/shape');
+const NodeosCommand = require('./cli-commands/nodeos');
 const DeployCommand = require('./cli-commands/deploy');
 const CompileCommand = require('./cli-commands/compile');
 
+const StartCommand = require('./cli-commands/nodeos/subcommands/start');
```

Comment:
Sub commands should be managed by the parent commands, not here

Reply:
updated